v1.0.0 Release (#26)
aerospike/php-client: README.md

### Setting up the Aerospike client connection manager: 

> #### Installing up the dependencies and Running Aerospike Connection manager

Reviewer: ```suggestion
#### Installing the dependencies and Running the Aerospike Connection manager
```

## Documentation

> * Reference documnetaion can be found [here] (https://aerospike.github.io/php-client/)

Reviewer: ```suggestion
* Reference documentation can be found [here] (https://aerospike.github.io/php-client/)
```

* Aerospike Documentation can be found [here](https://aerospike.com/docs/)

## Issues

> If there are any bugs, feature requests or feedback -> please create an issue on [GitHub](https://github.com/aerospike/php-client/issues). We will review it. 

Reviewer: ```suggestion
If there are any bugs, feature requests or feedback -> please create an issue on [GitHub](https://github.com/aerospike/php-client/issues). Issues will be regularly reviewed by the Aerospike Client Engineering Team.
```
